dist/joii.js

```diff
@@ -778,6 +787,19 @@
                 case 'immutable':
                     metadata.is_read_only = true;
                     break;
+                case 'SerializableAttribute':
```

Comment:
Yeah, sounds good!

I would go with `serializable` though, since it describes a modifier, not a command.

There's also this convention (we use in the office), that recommends to avoid negative-variables (e.g. something starting with "no" or "not"), since it easily creates confusion.